### ISSUE TITLE
Accessibility finding: All page content must be included in landmarks

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#139055]
### Checklist item

Automated-001	Axe DevTools has been executed for every page

### Severity level

4, Low. Consider fixing or exploring in 2-4 sprints post-launch.

### Page(s)/url(s)

https://staging.va.gov/my-va/submissions/9093f30e-0d09-43ff-a13f-087847063071?postLogin=true

### Details

Form viewer page - AXE is reporting that the skip link is not included in a landmark. This is a best practice violation. And this is true, but on VA.gov, the skiplink is in the same place, but it's NOT coming up as a best practice violation. So I'm not too worried about this, but wanted to let you know that it came up during the axe scan.

<img width="1021" height="324" alt="Image" src="https://github.com/user-attachments/assets/33443098-17fc-4480-9726-c479404787bc" />

### Reproduction steps

Run Axe on this page

### Proposed solution or next steps

The issue didn't come up on the other pages, so I'd recommend comparing the placement of the skiplink to see if there's something you should change here.

**Comment by rileyorr** [comment]
The team reviewed this issue on April 15th and concluded this is a platform level issue

**Comment by rileyorr** [comment]
I'm going to unassign myself from this ticket; you can tag me directly or reach my team in #benefits-employee-experience with questions or updates on this work!
